### PR TITLE
feat(service): Add services for some games

### PR DESCRIPTION
**Pull request by NikolasK-source**
- 0 A.D.
- Anno 1602
- Anno 1800
- Sid Meier's Civilization IV
- Sid Meier's Civilization V
- Factorio
- Minecraft
- Need for Speed: Most Wanted (2005)
- The Settlers History Collection
- Stellaris
- Stronghold Crusader
- SuperTuxKart
- Terraria
- Zero-K

### CODE REVIEW

**Review by erig0 — COMMENTED** [review]
Some of the services have a LOT of ports open. Can you point to docs for the list of ports needed?

Additionally please update `po/POTFILES.in` and `config/Makefile.am`.

**Comment by NikolasK-source** [comment]
Here are sources for the required ports:

- https://www.pcgamingwiki.com/wiki/0_A.D.#Ports
- https://1602.annowiki.de/index.php?title=Mehrspielermodus
- https://www.ubisoft.com/en-us/help/anno-1800/connectivity-and-performance/article/forwarding-ports-for-anno-1800/000061883
- https://portforward.com/civilization-iv/
- https://portforward.com/sid-meiers-civilization-v/
- https://wiki.factorio.com/Multiplayer
- https://www.hostinger.com/tutorials/how-to-port-forward-a-minecraft-server
- https://portforward.com/need-for-speed-most-wanted/
- https://www.ubisoft.com/en-us/help/the-settlers-history-collection/article/troubleshooting-connectivity-issues-in-the-settlers-history-collection-pc/000065836
- https://portforward.com/stellaris/
- https://portforward.com/stronghold-crusader/
- https://github.com/supertuxkart/stk-code/blob/master/NETWORKING.md
- https://portforward.com/terraria-server/
- https://portforward.com/zero-k/

**Comment by erig0** [comment]
Hi, I split this commit into a separate commit or each game. I also did some renaming.

**Are all these need to _host_ games?** In general, service are only needed to host a game.

File: config/services/civilization-iv.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    installment of the Civilization series.
+  </description>
+  <port port="2033" protocol="tcp"/>
+  <port port="2056" protocol="tcp"/>
+  <port port="3783" protocol="tcp"/>
+  <port port="6500" protocol="tcp"/>
+  <port port="6667" protocol="tcp"/>
+  <port port="28900" protocol="tcp"/>
+  <port port="29900-29901" protocol="tcp"/>
+  <port port="2033" protocol="udp"/>
```
Comment (erig0) [inline config/services/civilization-iv.xml:15]: I think this should be [2056][1], not 2033. Do you agree?
Although, [this forum][2] indicates 2033 and 2056.

[1]: https://portforward.com/civilization-iv/
[2]: https://forums.civfanatics.com/threads/unofficial-civ4-gamespy-multiplayer-firewall-router-port-forwarding-guide.149874/

**Review by erig0 — CHANGES_REQUESTED** [review]
Otherwise lgtm.

File: config/services/civilization-iv.xml (hunk 1 above)
Comment (NikolasK-source) [reply config/services/civilization-iv.xml:15]: According to [this site](https://www.pcgamingwiki.com/wiki/Sid_Meier%27s_Civilization_IV), neither 2033 nor 2056 udp is required.
With only 2033 it worked the last time I tested it. Unfortunately I currently don't have the possibility to test it without port 2033 opened.

**Review by erig0 — APPROVED** [review]

**Pull request merged**